src/configurator.rb

```diff
@@ -173,7 +175,25 @@ def determine_default(question_default:, old_answer:)
           end
         end
       end
-    end
 
+      def required_answers_check
```

Comment:
A better name here might document the intent of what we're doing - possibly something like `ensure_answer_given`?